app/webpack/components/authentication/sign_in.vue

```diff
@@ -230,6 +247,9 @@ export default {
           })
           .finally(() => {
             vm.loading = false;
+          })
+          .catch(error => {
```

Comment:
Just from a purely stylistic point of view, could you move the `.catch` to before the `.finally`?

Reply:
Done!